### ISSUE TITLE
Tooltip isn't triggered on focus event

### ISSUE THREAD

**Issue by devilcoders** [tvkhoa/react-tippy#14]
Tootlip `<div>` tag doesn't have `tabindex` attribute, so it's not receiving `focus` event when you use keyboard navigation.

**Comment by tvkhoa** [comment]
Thanks @devilcoders ,
Just added `tabIndex` prop in version `1.2.1`

**Comment by blindfish3** [comment]
Couldn't you automatically set tabIndex to 0 if trigger contains 'focus'?  Seems odd to have to do this manually to make focus work...

Also the trigger on focus behaviour looks a little flaky to me when combined with click trigger:

```
<Tooltip title="foo" trigger="focus click" tabIndex={0}><label>Foo Bar</label></Tooltip>
```

The initial click (when the element doesn't have focus) doesn't open the tooltip when I try this.   Looks to me like when you click you give the element focus and it starts opening the tooltip; but then registers the click which closes it.  Subsequent clicks (when the element *does* have focus) work as expected.

**Comment by alex-shatalov** [comment]
if i trigger focus event on element tooltip does not shows but when i press tab button its okey, is it normal?

**Comment by ChristopherVaccaro** [comment]
> if i trigger focus event on element tooltip does not shows but when i press tab button its okey, is it normal?

Same exact issue. I essentially need to press tab key twice to get the tooltip to show. Wish this wasn't the case along with the tabIndex working by default instead of having to define it. Best tooltip component around but can't use it without accessibility functioning
